mail_print/views/mail_message_report.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <template id="mail_message">
+            <t t-call="report.html_container">
+                <t t-foreach="docs" t-as="o">
+                    <t t-call="report.internal_layout">
+                        <div class="page">
+                            <style>
+                                .table1 {width: 50%}
+                                .table2 {width: 100%}
+                                .font {font-family: Helvetica, sans-serif;font-size: 10pt}
+                                .bold {font-weight: bold}
+                                .space {padding-right: 0.3em;}
+                            </style>
+                            <table class="table1 table-condensed">
+                                <tr>
+                                    <td class="font bold"><span>From:</span></td>
```

Comment:
consider using the `th` tag is you want to signify some cell is a column/row header